sdk/resourcemanager/azure-resourcemanager-containerservice/src/test/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterAgentPoolImplTests.java

```diff
@@ -59,7 +59,7 @@ void testGetAgentPoolInner() throws Exception {
         AgentPoolInner agentPoolInner = impl.getAgentPoolInner();
         for (Method method : agentPoolInner.getClass().getDeclaredMethods()) {
             String name = method.getName();
-            if (mockValues.containsKey(name)) {
+            if (mockValues.containsKey(name) && !"podIpAllocationMode".equals(name)) {
```

Comment:
Update KubernetesClusterAgentPoolImpl instead

Reply:
All added properties in AgentPoolInner.java need to be updated to this Impl

Reply:
Has been fixed by the commit#[ac5c14a](https://github.com/Azure/azure-sdk-for-java/pull/45642/commits/ac5c14a1c989e5dfd25693a2e1a62e4df8e7a90b)

Reply:
Add all new properties